pandas/core/arrays/categorical.py

```diff
@@ -1921,7 +1928,7 @@ def _get_repr(self, length=True, na_rep="NaN", footer=True) -> str:
         from pandas.io.formats import format as fmt
 
         formatter = fmt.CategoricalFormatter(
-            self, length=length, na_rep=na_rep, footer=footer
+            self, length=length, na_rep=na_rep, footer=footer, quoting=QUOTE_NONNUMERIC
```

Comment:
maybe don't need to pass this through here, since CategoricalFormatter should always be QUOTE_NONNUMERIC